Source/Common/Include/latticearchive.h

```diff
@@ -23,7 +23,7 @@
 #include <algorithm> // for find()
 #include "simplesenonehmm.h"
 #include "Matrix.h"
-
+#include <set>
```

Comment:
Do we need them sorted? Otherwise we can use unordered_set

Reply:
change to unordered set